test/stdlib/symbol-visibility-linux.test-sh

```diff
@@ -7,510 +7,39 @@
 // NOTE: this test is fragile.  It is dependent on the specifics of the GNU C++
 // runtime.  The C++ headers from the runtime explicitly force the weak symbols
 // to be publicly visible without allowing the user to control the visibility.
-// We explicitly filter out the ones that we see being leaked after manually
-// validating that they are being leaked because of the forceful nature of
-// libstdc++.
+// We filter out all leaked ::std:: symbols and explicitly filter out any
+// remaining symbols after after manually validating that they are being leaked
+// because of the forceful nature of libstdc++.
+//
+// To expand the `sed`:
+//   - `_Z` mangling prefix
+//   - `GTt` transaction entry point
+//   - `Z` couldn't find this, but seems to occur frequently
```

Comment:
Ah, this is scope encoding. Will update after smoke test + toolchain has run. This would also mean it should go before GTt.